<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewFolder</class>
 <widget class="QDialog" name="DialogNewFolder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建文件夹</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditName">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>描述</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="textEditBrief">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>添加文件夹描述...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>42</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>44</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewFolder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewFolder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
